mindsdb/__main__.py

```diff
@@ -29,9 +29,13 @@
 from mindsdb.utilities.context import context as ctx
 from mindsdb.utilities.auth import register_oauth_client, get_aws_meta_data
 
-import torch.multiprocessing as mp
 try:
+    import torch.multiprocessing as mp
     mp.set_start_method('spawn')
+    os.environ["MINDSDB_USE_TORCH"] = "1"
+except (ImportError, ModuleNotFoundError):
+    log.logger.info("torch is not installed. torch.multiprocessing won't be used")
+    import multiprocessing as mp
```

Comment:
NOTE: There's a bunch of commands below in "init" that use mp.* commands, which means they will fail I believe.  We need them to find alternatives.